frontend/src/OneBuilding/BuildingItem.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
```

Comment:
we have a `buildingItem` component in the `buildings` folder too, what is this one for? isn't the purpose of these 2 are exactly the same? if the goal is the same, please refactor that one, and use it, don't duplicate. if it's not the same, please explain what's the difference and what's the goal of these because it's not obvious for me

Reply:
They are not exactly the same. The one in the buildings folder is clickable and Sherry has modified it with some extra CSS styles; the one here is just a static component which used for displaying icon and texts only, and it has different CSS styles :^)

Reply:
we use the buildingItem in the buildings folder for listing the user's buildings, coming from the backend as an array.

this one is for the popup modal?

Reply:
yes, its for the popup modal, I guess I will rename it like 'popupItem.jsx' instead of 'buildingItem.jsx'

Reply:
great, thanks!